corehq/apps/hqcase/utils.py

```diff
@@ -210,7 +225,17 @@ def update_case(
     )
 
 
-def bulk_update_cases(domain, case_changes, device_id, xmlns=None):
+type CaseID = str
```

Comment:
[The `type` statement](https://docs.python.org/3/reference/simple_stmts.html#type) declares an alias, for simplifying types and giving them self-explanatory names. (Python 3.12+)